main/src/library/Concurrent/Channel.flix

```diff
@@ -67,59 +72,50 @@ namespace Concurrent/Channel {
     /// conditions in `waitingPutters` comes from locks created by the get
     /// function or the select function.
     ///
```

Comment:
As always, I would ask for some documentation here to explain that Mpmc now consists of two parts.